### PR TITLE
fix(design-tokens): text5 color is now identical to text (blend=100%)

### PR DESCRIPTION
**Pull request by lukelooker**
Make text5 = text

## Developer Checklist [ℹ️](https://github.com/looker-open-source/components/blob/main//CONTRIBUTING.md#developer-checklist)

- [ ] 🖼 Image Snapshot coverage

### CODE REVIEW

File: packages/design-tokens/src/color/blendPoints.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@
  */
 
 export type TextColorLevels = 0 | 1 | 2 | 3 | 4 | 5
-export const textBlends = [45, 65, 78, 88, 95, 99]
```
Comment (lukelooker) [inline packages/design-tokens/src/color/blendPoints.ts:28]: This is the _actual_ change.

Comment (lukelooker) [reply packages/design-tokens/src/color/blendPoints.ts:28]: Changes text5 from being `95%` to `100%` - upper bound here wasn't in use (mistake)

File: packages/design-tokens/src/color/utils/mixColors.ts [hunk 2]
```diff
@@ -30,4 +30,5 @@ export const mixColors = (
   mixAmount: number,
   foreground: string,
   background: string
-) => mix(mixAmount / 100, foreground, background)
+) =>
+  mixAmount === 100 ? foreground : mix(mixAmount / 100, foreground, background)
```
Comment (lukelooker) [inline packages/design-tokens/src/color/utils/mixColors.ts:34]: No need to blend if color mix is 100%

**Review by jhardy — APPROVED** [review]
👍

**Pull request merged**